### PR TITLE
CHKT-1394 Silence "low-hanging fruit" warnings

### PR DESCRIPTION
**Pull request by borisprimer**

### CODE REVIEW

File: Internal/Debug App/Sources/Network/Networking.swift [hunk 1]
```diff
@@ -96,11 +96,10 @@ class Networking {
                         
         msg += "Headers:\n\(request.allHTTPHeaderFields ?? [:])\n"
                 
-        if let body = body {
+        if let body = body,
+           let bodyJson = try? JSONSerialization.jsonObject(with: body, options: .allowFragments){
             request.httpBody = body
-            
-            let bodyJson = try? JSONSerialization.jsonObject(with: body, options: .allowFragments)
-            msg += "Body:\n\(bodyJson ?? [:])\n"
```
Comment (EvansPie) [inline Internal/Debug App/Sources/Network/Networking.swift:103]: This was written this way as to always have console log and know what's happening, even if the body was `nil`.

File: Internal/Debug App/Sources/View Controllers/MerchantResultViewController.swift [hunk 2]
```diff
@@ -42,11 +42,7 @@ class MerchantResultViewController: UIViewController {
         if logs.count > 0 {
             if let data = try? JSONSerialization.data(withJSONObject: logs) {
                 if let prettyNSStr = data.prettyPrintedJSONString {
-                    if let prettyStr = prettyNSStr as? String {
```
Comment (EvansPie) [inline Internal/Debug App/Sources/View Controllers/MerchantResultViewController.swift:45]: This is needed in the case of some failures of WebDriverIO (used in UI tests).

File: Internal/Debug App/Sources/View Controllers/MerchantResultViewController.swift [hunk 3]
```diff
@@ -57,14 +53,11 @@ class MerchantResultViewController: UIViewController {
...
         }
 
+
         if let checkoutData {
             if let data = try? JSONEncoder().encode(checkoutData) {
                 if let prettyNSStr = data.prettyPrintedJSONString {
-                    if let prettyStr = prettyNSStr as? String {
-                        responseTextView.text = prettyStr
-                    } else {
-                        responseTextView.text = "[\"Failed to case NSString to String [WebDriverIO]\"]"
```
Comment (EvansPie) [inline Internal/Debug App/Sources/View Controllers/MerchantResultViewController.swift:66]: Same as above

File: Internal/Debug App/Sources/Network/Networking.swift [hunk 4]
```diff
@@ -262,7 +261,6 @@ class Networking {
         let url = environment.baseUrl.appendingPathComponent("/api/client-session")
 
         var bodyData: Data!
-        var headers: [String: String]?
```
Comment (EvansPie) [inline Internal/Debug App/Sources/Network/Networking.swift:265]: The reason we had those here is for when we want to test something in particular and need to pass some extra headers.

**Review by EvansPie — CHANGES_REQUESTED** [review]

File: Internal/Debug App/Sources/Network/Networking.swift (hunk 1 above)
Comment (borisprimer) [reply Internal/Debug App/Sources/Network/Networking.swift:103]: OK, reversed

File: Internal/Debug App/Sources/View Controllers/MerchantResultViewController.swift (hunk 2 above)
Comment (borisprimer) [reply Internal/Debug App/Sources/View Controllers/MerchantResultViewController.swift:45]: reversed.

File: Internal/Debug App/Sources/View Controllers/MerchantResultViewController.swift (hunk 3 above)
Comment (borisprimer) [reply Internal/Debug App/Sources/View Controllers/MerchantResultViewController.swift:66]: reversed.

**Review by EvansPie — APPROVED** [review]

**Pull request merged**